.gitignore

```diff
@@ -1 +1,6 @@
 /target
+/Cargo.lock
```

Comment:
Lock files should be included in the version control right? They allow loading the exact dependency versions which are useful when there are breaking changes in the dependencies of dependencies that are referenced using a git branch (so without a lock file somebody doing a fresh install 1 year later would pull the latest versions which then won't compile).